content/00.front-matter.md

```diff
@@ -5,7 +5,7 @@
 ##}
 
 {## Uncomment & edit the following line to reference to a preprinted or published version of the manuscript.
-_A DOI-citable version of this manuscript is available at <https://doi.org/DOI_HERE>_.
+_A DOI-citable version of this manuscript is available at <https://arXiv:1910.11619 >_.
```

Comment:
This line is actually commented out so it won't show up on the output manuscript. We also probably don't want to change it because it's for pointing to a published version of a manuscript... which for try-manubot will never exist.

Reply:
ok,thanks